Formula/unnethack.rb

```diff
@@ -1,8 +1,8 @@
 class Unnethack < Formula
   desc "Fork of Nethack"
   homepage "https://sourceforge.net/projects/unnethack/"
```

Comment:
maybe we should also update the homepage to the github one?

Reply:
Oh yeah we should

Reply:
Cool!